package/yast2-bootloader.changes

```diff
@@ -1,10 +1,18 @@
 -------------------------------------------------------------------
+Mon May  2 10:24:17 CEST 2016 - schubi@suse.de
+
+- Fixed nil in custom_devices. [[]].compact! is returning nil.
```

Comment:
Hm, as far I see it is returning also array in array too. Otherwise I cannot reproduce this error that custom_devices is nil. Could it be that we have different data types in this array and have a complete different bug here which has been found by the "reject" call ? Will check again......